### PR TITLE
Update setup.py to load requirements from file and fix example in example.py

### PR DESCRIPTION
**Pull request by dimaglushkov**
Hello and thanks for a great project! 
I'm currently using this tool at my script to automatically get problem-related info to my local repo, create a code template for a solution, and automatically update a table with solved problems in README.md.
With this PR I've tried to fix two open issues

### CODE REVIEW

**Comment by prius** [comment]
Hi! Thanks for working on that! And so sorry for getting back to you so late. The recent events just knocked me down for a week and made me unable to work at all.

Anyway, your changes look good. Give me a second to verify them on my machine, and then I'll merge the repo and bump the version.

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,8 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["urllib3 >= 1.15", "six >= 1.10", "certifi", "python-dateutil", "requests"]
+with open("requirements.txt") as requirements:
+    REQUIRES = [req.replace('\n', '') for req in requirements.readlines()]
```
Comment (prius) [inline setup.py:30]: There should be double quotes here in accordance with PEP8. Just run this file through black auto-formatter to make it compliant.
```suggestion
    REQUIRES = [req.replace("\n", "") for req in requirements.readlines()]
```

**Comment by prius** [comment]
Added one comment and tested both build and modified example.py. Everything LGTM as long as the comment is addressed.

**Comment by dimaglushkov** [comment]
Ah, that's must be some kind of a bad habit coz python isn't my main language :/
I've applied your suggestion and I believe that PR is safe to merge now
At this horrible time developing stuff is the only thing which keeps me from losing my mind
Anyway, stay safe

**Pull request merged**